packages/react/src/components/Badge/index.test.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import Badge, { BadgeLabel } from '../../../src/components/Badge';
+import axe from '../../axe';
+
+test('renders children', () => {
+  render(
+    <Badge>
+      <BadgeLabel>Label:</BadgeLabel> value
+    </Badge>
+  );
+  expect(screen.getByText('Label:')).toBeInTheDocument();
+});
+
+test('passes classNames through', () => {
+  render(
+    <Badge className="baz">
+      <BadgeLabel className="jazz">Badge:</BadgeLabel>
+      hi
+    </Badge>
+  );
+  expect(screen.getByText('hi')).toHaveClass('baz');
+  expect(screen.getByText('Badge:')).toHaveClass('jazz');
```

Comment:
This should also include testing for the `.Badge` and `.Badge__Label` classes to ensure they're not being overridden by `className`.